modules/heat_transfer/test/tests/gap_perfect_transfer/gap_perfect_transfer_rz.i

```diff
@@ -0,0 +1,139 @@
+#
+# 1-D Gap Perfect Heat Transfer
```

Comment:
do we need this RZ test? There's nothing special in RZ in the new object?

Reply:
Maybe not. I added an RZ test because this is how we found that the penalty parameter was not controllable via the Action when this object was in BISON. Maybe instead of this test as it is (the increase in penalty doesn't change the answer) I should add a coupled mechanics/heat transfer test that uses this object in the `combined` module. Then the need for a higher penalty becomes evident.

Reply:
as you want if you think there's a need for a coupled example
I d remove the RZ though

Reply:
Since the RZ test was mostly to check that the `penalty` parameter could be set through the action, the addition of the error check for setting parameter when not using `GapPerfectConductance` can serve the same purpose. I'll remove this in favor of a RunException test that ensures the correct paramError is returned.